.github/workflows/main.yml

```diff
@@ -0,0 +1,54 @@
+name: Run Tests
+
+on:
+  push:
+    branches: [ develop ]
+  pull_request:
+    branches: [ develop ]
+
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7]
```

Comment:
Let's try 3.8 again!

Reply:
1 testcase is failed for python 3.8-> https://github.com/anitab-org/portal/pull/632/checks?check_run_id=945743460#step:7:22

Reply:
I am not sure why this specific test was successful all this time, because [`assertRaises()`](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRaises) takes a callable, could you fix https://github.com/anitab-org/portal/blob/develop/systers_portal/meetup/tests/test_forms.py#L81 so that `form.clean_time` callable is passed, and not the executed function?

Reply:
Any updates on this?

Reply:
On it.

Reply:
There were some errors in test cases, which I had fixed.